src/main/java/mil/dds/anet/resources/LocationResource.java

```diff
@@ -99,4 +99,41 @@ public Integer updateLocation(@GraphQLRootContext Map<String, Object> context,
     return numRows;
   }
 
+  @GraphQLMutation(name = "mergeLocation")
+  public Location mergeLocation(@GraphQLRootContext Map<String, Object> context,
+      @GraphQLArgument(name = "loserLocationUuid") String loserLocationUuid,
+      @GraphQLArgument(name = "loserLocationUuid2") String loserLocationUuid2,
```

Comment:
I believe in all cases, we still want to have a winner and a loser. In a lot of cases, we want to retain the uuid of the more prominent of the two locations, so that external links to that uuid remain functional. 
I may be wrong, but I can't think of a use case where we would want 2 losers and a merged object with a new uuid.

Reply:
I thought of continuing with the uuid info of the prominent location, but as a result I continued with the logic of 2 objects merging and a new object is formed. I will turn it into a structure that is a winner and a looser.